<?xml version="1.0"?>
<launch>

	<arg name="robot" />

	<rosparam file="$(find cob_controller_configuration_gazebo)/ros/controller/arm_controller_$(arg robot).yaml" command="load"/>

	<node name="arm_controller_spawner" pkg="pr2_controller_manager" type="spawner" args="arm_controller" />
	
	<group ns="arm_controller">
		<node name="arm_joint_trajectory_action_node" pkg="joint_trajectory_action" type="joint_trajectory_action" />
	</group>

</launch>
